LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -4666,3 +4666,281 @@ TacticalText="<Bullet/> Плазмобластер выпускает разру
+
+[Vektor_CG X2WeaponTemplate]
+FriendlyName="\"Flatline\" Crossbow"
+FriendlyNamePlural="\"Flatline\" Crossbows"
+BriefSummary="Наш финальный дизайн арбалета с самыми передовыми технологиями. Болты этого арбалета содержат поле микроплазмы, у меньшая сопротивление воздуха и легко проникая броню"
+TacticalText="<Bullet/> Оружие Гаусса автоматически проникает (игнорирует) 1 броню.<br/><Bullet/> Оружия Жнецов очень точные вблизи, также хороши и на дальних дистанциях.\n<Bullet/> Многие Жнецы предпочитают композитный арбалет огнестрельному оружию. Действует подобно винтовки Вектор с технологиями Гаусса.\n<Bullet/> Исользуйте найденные модификации чтобы повысить тактическую  эффективность арбалета."
+AbilityDescName="crossbow"
+
+[VektorRifle_BM X2WeaponTemplate]
+TacticalText="<Bullet/> Плазменное оружие повреждает цель так, что следующие удары по ней будут наносить +1 урон.\n<Bullet/> Самый лучший вариант винтовок Вектор из арсенала ХКОМ.\n<Bullet/> Оружия Жнецов очень точные вблизи, также хороши и на дальних дистанциях.\n<Bullet/> Исользуйте найденные модификации чтобы повысить тактическую  эффективность Shadow Lance's tactical effectiveness."
+
+[SPARKChaingun X2WeaponTemplate]
+FriendlyName="Тяжелый пулемет"
+FriendlyNamePlural="Тяжелые пулеметы"
+BriefSummary="Массивный пулемет с разрушительными противопехотными возможностями."
+TacticalText="<Bullet/> Оружие сделанное по чертежам Рэймонда Шень ещё со времен первого вторжения инопланетян.\n<Bullet/> Функциональность идентична базовой автопушке "
+AbilityDescName="Тяжелый пулемет"
+
+[SparkRifle_LS X2WeaponTemplate]
+FriendlyName="Рельсотронная пушка"
+FriendlyNamePlural="Рельсотронн пушки"
+BriefSummary="Рельсотрон стреляет снарядами большого калибра разнося всё вокруг."
+TacticalText="<Bullet/> Laser weapons receive bonus aim.\n<Bullet/> Рельсотронная пушка is the laser equivalent of the Autocannon."
+AbilityDescName="Рельсотронная пушка"
+
+[SparkRifle_CG X2WeaponTemplate]
+FriendlyName="Дезинтегратор частиц"
+FriendlyNamePlural="Дезинтеграторы частиц"
+BriefSummary="Дезинтеграторы используют многократное ускорение сфокусированных лучей для поражения целей на любой дистанции." 
+TacticalText="<Bullet/> Оружие Гаусса автоматически проникает (игнорирует) 1 броню.<br/><Bullet/> Дезинтегратор частиц is the Coil equivalent of the Autocannon."
+AbilityDescName="Дезинтегратор частиц"
+
+; End Translated
+
+
+
+
+; LWOTC Translated
+BriefSummary="Солдает получает <Ability:TRIGGER_BSC_AIM_BONUS/> бонус к точности ответных выстрелов из основого оружия."
+; End Translated
+
+; LWOTC Translated
+BriefSummary="Солдает получает <Ability:TRIGGER_ADV_AIM_BONUS/> бонус к точности ответных выстрелов из основого оружия."
+; End Translated
+
+; LWOTC Translated
+BriefSummary="Солдает получает <Ability:TRIGGER_SUP_AIM_BONUS/> бонус к точности ответных выстрелов из основого оружия."
+; End Translated
+
+
+;LWOTC needs translation
```

Comment:
I assume this is supposed to be "LWOTC Translated"?

Reply:
yes